src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -111,15 +111,29 @@ function mergeStandByTransactionsInRange({
   standByTransactions: TokenTransaction[]
   currentCursor?: number
 }): TokenTransaction[] {
-  if (transactions.length === 0) return []
+  /**
+   * If the data from the first page is empty - there's no successful transactions in the wallet.
+   * Maybe the user executed a single transaction, it failed and now it's in the standByTransactions.
+   * In this case we need to show whatever we've got in standByTransactions, until we have some
+   * paginated data to merge it with.
+   */
+  const isFirstPage = currentCursor === FIRST_PAGE_TIMESTAMP
```

Comment:
Same comment from the other PR; I think the cursor semantics here need to change. With the [current implementation](https://github.com/valora-inc/blockchain-api/blob/main/src/cloudFunctions/endpoints/getWalletTransactions/index.ts#L121) in `blockchain-api`, the `hasPreviousPage` field in the response is only set when the client requests a specific page (the reasoning behind this is that if the client is requesting a particular page, it must have gotten that page cursor from an earlier page, so we can assume that previous pages exist). The _absense_ of the `hasPreviousPage` field in the response can then be used to know if the response is in fact the first (newest) page.